tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2878,11 +2977,13 @@ nat:
 #######################################
 #####             ospf            #####
 #######################################
+
 ospf:
   skip:
-    reason: "Neighbor type must be sonic, skip in PR testing"
+    reason: "Neighbor type must be sonic, skip in PR testing and Test is skipped for Mellanox SN5640 platform and for  Arista-7060X6-64PE-B-* TH5 SKUs "
     conditions:
       - "asic_type in ['vs']"
+      - "hwsku in ['Arista-7060X6-64PE-B-C448O16','Arista-7060X6-64PE-B-C512S2','Mellanox-SN5640-C512S2','Mellanox-SN5640-C448O16']"
```

Comment:
vs couldn't be in those hwskus. Those 2 conditions are or, not and. Please add the operator "or"